aya-obj/src/obj.rs

```diff
@@ -987,6 +1006,14 @@ pub enum ParseError {
     NoBTF,
 }
 
+/// Invalid bindings to the bpf type from the parsed/received value.
+pub struct InvalidTypeBinding<T> {
+    /// The bpf type to bind to.
+    pub binding_type: String,
```

Comment:
This is not being read anywhere I think, so I would remove it? Doesn't matter
much because in practice i don't think it's ever instantiated, but avoid
allocations etc :P

Reply:
Make sense. 🫡